File: Androidv0/app/src/main/java/tn/esprit/nascar/Models/Reclamation.kt
package tn.esprit.nascar.Models

class Reclamation(
    var id: Int,
    var title: String,
    var type: String,
    var description: String,
    var img: String,
    var id_user: Int
) {

    override fun toString(): String {
        return "Reclamation(id=$id, title='$title', type='$type', description='$description', img='$img')"
    }
}

File: backend/backend-main/models/Reclamation.js
const mongoose = require('mongoose');


const Reclamation = mongoose.model('Reclamation',{
    title: {
        type:String,
        require : true
    },
    type: {
        type:String,
        require : true
    },
    description: {
        type: String,
        // require : true
    },
    img: {
        type: string,
        // require : true
    },
    id_user: {
        type:integer,
    }
})

module.exports = Reclamation;